emcc.py

```diff
@@ -1623,6 +1623,10 @@ def default_setting(name, new_default):
   if settings.DYNCALLS and not settings.MINIMAL_RUNTIME:
     settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$dynCall']
 
+  if not settings.BOOTSTRAPPING_STRUCT_INFO:
+    # Include the built-in runtime to call function pointers from JS.
+    settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$wbind', '$wbindArray', '$setWasmTableEntry']
```

Comment:
Ah.. I missed this originally, we unconditionally include there.   As I said in my other command maybe we can make addFunctionWasm into a library function and avoid this here?  I guess that could be followup.

BTW it looks like `$wbindArray` might not be needed here since all its caller are library function so can include it in their `__deps`.

Reply:
yeah, looks like can drop wbindArray here.